server/index.ts

```diff
@@ -0,0 +1,114 @@
+import { Alchemy, Network } from 'alchemy-sdk';
+import { NftToolServerRequest, NftToolServerResponse, NftToolServerResponseData } from '../src/types/index';
+
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+import cors from 'cors';
+import express from 'express';
+
+const app = express();
+
+app.use(cors());
+app.use(express.json());
+
+/**
+ * Create endpoint for Alchemy API
+ */
+app.post('/alchemy', async (req, res) => {
+  try {
+    const { network, contractAddress, tokenId }: NftToolServerRequest = req.body;
+
+    /**
+     * Validate request body
+     */
+    if (!network) {
+      throw new Error('«network» is required');
```

Comment:
Please, don't use these quotation marks in English) They use " or '.
https://en.wikipedia.org/wiki/Quotation_mark#Summary_table